packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,41 @@
+import { SmallText } from "@stenajs-webui/core";
+import * as React from "react";
+import { Popover, PopoverProps } from "../popover/Popover";
+import { defaultTooltipTheme, TooltipTheme } from "./TooltipTheme";
+
+export interface TooltipProps extends Omit<PopoverProps, "content"> {
+  type?: TooltipInfoType;
+  label: string;
+  theme?: TooltipTheme;
+}
+
+type TooltipInfoType = "info" | "warning" | "error";
```

Comment:
In Freight Portal, we have
```
  Confirmed,
  HardWarning,
  SoftWarning,
  UrgentInfo,
  RegularInfo,
```
Sync with Chris and Ozcar?

Reply:
Skipping this for now, and we can make a generic framework-wide solution for notice levels.